Core/Random.cc

```diff
@@ -35,6 +35,10 @@ namespace LogCabin {
 namespace Core {
```

Comment:
Please add copyright statement after line 2 (the copyright owner would be Scale Computing unless you have explicit permission from them to retain copyright).